core/src/autogluon/core/metrics/__init__.py

```diff
@@ -454,6 +541,11 @@ def get_metric(metric, problem_type=None, metric_type=None) -> Scorer:
             # Requires mxnet
             from .softclass_metrics import soft_log_loss
             return soft_log_loss
+        elif metric == 'pinball_loss':
+            if problem_type is not None and problem_type is not QUANTILE:
```

Comment:
`if problem_type is not None and problem_type != QUANTILE` is better

Reply:
This will be fixed.